pyconbalkan/timetable/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from pyconbalkan.core.models import ActiveModel
+from pyconbalkan.speaker.models import Speaker
+from django.db.models import CASCADE
+
+
+class Timetable(ActiveModel):
+    title = models.CharField(null=True, blank=True, max_length=100)
+    date = models.DateField(null=True, blank=True)
+
```

Comment:
@nadaj please, also add `__string__` representation for all the models:
- Timetable: 
> `title`
- Presentation: 
> [`type`] `title`
- Room: 
> `name`
- Slot: 
> From `from_date` to `to_date`

Thanks :)